### ISSUE TITLE
The ssh client now hangs

### ISSUE THREAD

**Issue by ahyattdev** [darlinghq/darling#303]
Labeled as a regression because this used to work. 

```
Darling [~]$ ssh andyhyatt@172.16.0.3                                  
Segmentation fault: 11 (core dumped)
```

This issue is on hold until we have a proper debugger working under Darling.

**Comment by LubosD** [comment]
Does not crash for me, it rather dies on a missing syscall:
```
Unimplemented syscall (243)
initgroups root 0: Function not implemented
```

**Comment by LubosD** [comment]
Please check if it works for you now.

**Comment by ahyattdev** [comment]
It still segmentation faults, but not until after a correct password is entered at the prompt. Progress, I guess

**Comment by ahyattdev** [comment]
It now no longer crashes but instead hangs before the password
